include/deal.II/lac/precondition.h

```diff
@@ -1615,7 +1615,8 @@ class PreconditionChebyshev : public Subscriptor
     /**
      * This determines the degree of the Chebyshev polynomial. The degree of
      * the polynomial gives the number of matrix-vector products to be
-     * performed for one application of the vmult() operation. Degree one
+     * performed for one application of the step() operation. During vmult()
+     * a matrix-vector product less is performed. Degree one
```

Comment:
"product less"? I don't understand what this means.

Reply:
I think you want to say 
```suggestion
     * the method performs `(degree-1)` matrix-vector products. Degree one
```

Reply:
Changed!